Berksfile

```diff
@@ -2,3 +2,5 @@ source 'https://supermarket.getchef.com'
 
 cookbook 'python-webapp-test', path: 'test/cookbooks/python-webapp-test'
 metadata
+cookbook 'git'
```

Comment:
Unless these are coming from somewhere other than supermarket, they don't need to be listed here as they are already listed in metadata.